ipc-agent-sdk/src/message/chain.rs

```diff
@@ -0,0 +1,16 @@
+// Copyright 2022-2023 Protocol Labs
+// SPDX-License-Identifier: MIT
+//! Contains the chain related RPC request and response messages
+
+use crate::message::CIDMap;
+use serde::{Deserialize, Serialize};
+use serde_json::Value;
+
+/// A simplified struct representing a `ChainHead` response that does not decode the `blocks` field.
+#[derive(Debug, Deserialize, Serialize)]
```

Comment:
The plan is to move all types here, right? Would you mind opening an issue explaining what needs to be moved to the `sdk` in a follow-up PR. Thanks!

Reply:
Added